zhaquirks/tuya/TS011F_plug.py

```diff
@@ -0,0 +1,112 @@
+"""TS011F plug."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Identify, OnOff, Ota, Scenes, Time
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODEL,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+class TuyaZBMetering(CustomCluster, Metering):
+    """TuyaZBMetering cluster implementation. This implementation divides the KwH for tuya devices"""
+
+    cluster_id = Metering.cluster_id
+    current_summ_delivered = 0x0000
+
+    def _update_attribute(self, attrid, value):
+        if attrid == self.current_summ_delivered:
+            value = value / 100
+        super()._update_attribute(attrid, value)
+
+class TuyaZBElectricalMeasurement(CustomCluster, ElectricalMeasurement):
+    """TuyaZBElectricalMeasurement cluster implementation. This implementation divides the Current for tuya devices"""
+
+    cluster_id = ElectricalMeasurement.cluster_id
+    rms_current = 0x0508
+
+    def _update_attribute(self, attrid, value):
+        if attrid == self.rms_current:
+            value = value / 1000
+        super()._update_attribute(attrid, value)        
```

Comment:
```suggestion
    AC_VOLTAGE_MULTIPLIER = 0x0600
    AC_VOLTAGE_DIVISOR = 0x0601
    
    _CONSTANT_ATTRIBUTES = {AC_VOLTAGE_MULTIPLIER: 1, AC_VOLTAGE_DIVISOR: 100}
```

Try just defining the divisor and multiplier

Reply:
Hi, not working anymore. Value isn't divided by 1000 anymore
AC_Voltage_Divisor should be 1000